pkgs/by-name/ne/nexusmods-app/update.bash

```diff
@@ -7,3 +7,6 @@ set -o errexit -o nounset -o pipefail
 package=nexusmods-app
 nix-update "$package"
 "$(nix-build --attr "$package".fetch-deps --no-out-link)"
+
+# Format using the nixfmt pinned in shell.nix
+nix-shell --run "nixfmt pkgs/by-name/${package:0:2}/$package/**"
```

Comment:
I wonder if we should maybe skip formatting dep.nix until we do something like https://github.com/NixOS/nixpkgs/pull/325053?  This will also lose formatting if someone runs `fetch-deps` directly.

Reply:
I think https://github.com/NixOS/nixpkgs/pull/326407 should catch anyone doing that since it enforces formatting for any files that were previously formatted.

But the PR you linked would make running nixfmt somewhat redundant, assuming there's no bugs in the implementation.

Still, I don't think it does any harm?

Reply:
Yeah, it's just unfortunate that running `fetch-deps` (e.g. via `maintainers/scripts/update-dotnet-lockfiles.nix`) will result in changes that fail tests.

There's no way to temporarily mark a file as 'generated' or otherwise cause it to be ignored by the format tests?

If not, it's not a big deal.  I think long-term we'll want to run `nixfmt` automatically in `fetch-deps` rather that depending on `nuget-to-nix` trying to hit what may be a moving target.

Reply:
> Yeah, it's just unfortunate that running `fetch-deps` (e.g. via `maintainers/scripts/update-dotnet-lockfiles.nix`) will result in changes that fail tests.

That's fair. I can drop reformatting the generated file from this PR then so we can focus on just trying to get the app to build & run!

> There's no way to temporarily mark a file as 'generated' or otherwise cause it to be ignored by the format tests?

I think that'd defeat the point of trying to keep the file formatted?

> I think long-term we'll want to run `nixfmt` automatically in `fetch-deps` rather that depending on `nuget-to-nix` trying to hit what may be a moving target.

Agreed, that sounds like the ideal long-term approach!

Reply:
Please remove this again. This is unecessary right now.